### PR TITLE
fix: umi external bug

### PR DESCRIPTION
**Pull request by ycjcl868**
##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

##### Description of change

`umi` shouldn't be external.

image

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

### CODE REVIEW

File: packages/umi-build-dev/src/getWebpackConfig.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default function(service, opts = {}) {
 
   if (ssr) {
     webpackConfig.externals = nodeExternals({
-      whitelist: /\.(css|less|sass|scss)$/,
+      whitelist: [/\.(css|less|sass|scss)$/, /^umi/],
```
Comment (sorrycc) [inline packages/umi-build-dev/src/getWebpackConfig.js]: `/^umi/` -> `/^umi\//`

Comment (ycjcl868) [reply packages/umi-build-dev/src/getWebpackConfig.js]: `import { Link } from 'umi'` 会被 external 掉

Comment (sorrycc) [reply packages/umi-build-dev/src/getWebpackConfig.js]: `/^umi\/?$/`，避免 umi-plugin-xx、umi-request、umi-test 等也被打进去。

Comment (ycjcl868) [reply packages/umi-build-dev/src/getWebpackConfig.js]: `/^umi(\/.*)?$/` 会增加 97kb(587 - 490)

Comment (sorrycc) [reply packages/umi-build-dev/src/getWebpackConfig.js]: server 不用管大小。

**Pull request merged**